Rakefile

```diff
@@ -33,6 +33,9 @@ protoc_jsonschema_version="73d5723"
 namespace :codegen do
   task :clean do
       sh "rm -rf #{gogo_dir}"
+      if Dir.exist?(toolchain_dir)
+          sh "chmod u+w -R #{toolchain_dir}"
```

Comment:
Quetsion: Would it make more sense to add the permissions in the task that generates the proto files?

Reply:
I'm new here but I think that this is a good way to go around this - in most cases we don't want to overwrite anything in the directory so it's readonly